cache/lru_cache.cc

```diff
@@ -351,7 +351,7 @@ Status LRUCacheShard::Insert(const Slice& key, uint32_t hash, void* value,
 
   e->value = value;
   e->deleter = deleter;
-  e->charge = charge;
+  e->charge = charge + sizeof(LRUHandle) - 1;
```

Comment:
It's more intuitive for `e->charge` to be the value users pass in. We can add this extra overhead every time this is used. We can even define a function LRUHandle::adjusted_charge() or something to make it easier to use it.

By the way, there are some places `charge` is still directly used in this function.